src/applications/hca/tests/reducers.unit.spec.js

```diff
@@ -94,50 +94,57 @@ describe('HCA Enrollment Status Reducer', () => {
   });
 
   describe('FETCH_ENROLLMENT_STATUS_FAILED', () => {
-    describe('regardless of the error codes', () => {
```

Comment:
Github is making this diff uglier than it really is. I actually deleted this old "regardless of the error" test because the behavior of the reducer has changed a little bit and all the other tests make the original test redundant.